### PR TITLE
Add send message SC example

### PR DESCRIPTION
**Pull request by Eitu33**

### CODE REVIEW

**Comment by Eitu33** [comment]
yarn.lock changes are there because the format has changed with the latest versions, thought I'd push it but maybe we want to keep the previous version?

File: package.json [hunk 1]
```diff
@@ -9,17 +9,19 @@
...
-    "assemblyscript": "^0.19.20",
+    "assemblyscript": "^0.19.23",
     "jest": "^27.4.5"
   },
   "dependencies": {
     "as-base58": "^0.1.1",
     "as-hmac-sha2": "^1.0.2",
     "json-as": "^0.2.6",
-    "massa-sc-std": "https://github.com/z80dev/massa-sc-std",
+    "massa-sc-std": "https://github.com/massalabs/massa-sc-std#5c90b49",
```
Comment (qdrn) [inline package.json:24]: I don't think you want to use a specific branch. Let's just wait for the `massa-sc-std` branch with the `send_message` to be merged.

**Review by qdrn — CHANGES_REQUESTED** [review]
You can also remove `.wat` and `.wasm.map` files as they are of no use.

**Comment by Eitu33** [comment]
@qdrn should we merge this as it is or wait for https://github.com/massalabs/massa-sc-std/pull/11 to be completed in order to use the new version tag?

**Comment by gregLibert** [comment]
@Eitu33 you should remove (add to gitignore + delete) wasm files also.

**Comment by Eitu33** [comment]
> @Eitu33 you should remove (add to gitignore + delete) wasm files also.

I don't agree, if someone wants a ready to go example SC wasm to test out something they have that possibility without needing to compile anything (which has a few prerequisites).

**Comment by yvan-sraka** [comment]
We can put `.WASM` files, and other binaries artifacts under Git LFS management, if we want to track them here :)

**Review by qdrn — APPROVED** [review]

**Pull request merged**